<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="OpenGL" name="openGLWidget"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionSettings"/>
   </widget>
   <widget class="QMenu" name="menuSidebar">
    <property name="title">
     <string>Sidebar</string>
    </property>
    <addaction name="actionShow_Hide"/>
   </widget>
   <widget class="QMenu" name="menuRecord">
    <property name="title">
     <string>Record</string>
    </property>
    <addaction name="actionTake_screenshoot"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSidebar"/>
   <addaction name="menuRecord"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionOpen">
   <property name="text">
    <string>Open...</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionShow_Hide">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Show/Hide</string>
   </property>
  </action>
  <action name="actionSettings">
   <property name="text">
    <string>Settings</string>
   </property>
  </action>
  <action name="actionTake_screenshoot">
   <property name="text">
    <string>Take screenshot</string>
   </property>
  </action>
  <action name="actionTake_GIF">
   <property name="text">
    <string>Take GIF</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>OpenGL</class>
   <extends>QOpenGLWidget</extends>
   <header>opengl.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
